engine/src/main/java/org/terasology/rendering/dag/RenderGraph.java

```diff
@@ -28,19 +28,14 @@ public RenderGraph() {
         nodes = Lists.newArrayList();
     }
 
-    public String add(Node node, String suggestedId) {
+    public String addNode(Node node, String suggestedId) {
         nodes.add(node);
         return null; // TODO: for instance if "blur" present make id "blur1" and return it
```

Comment:
I'd return suggestedId for the time being, but if you have reasons to keep it this way for the time being I'd be ok with it.